snippets/markdown.json

```diff
@@ -24,6 +24,15 @@
 		"description": "Site region shortcode"
 	},
 
+	"Gov region selector":{
+		"prefix": ";;gov-region",
+		"body": [
+			"{{< site-region region=\"gov\" >}}<div class=\"alert alert-warning\"> Enter summary here <a href=\"/getting_started/site\">Datadog site</a> ({{< region-param key=\"dd_site_name\" >}}).</div>",
+"{{< /site-region >}}"
```

Comment:
I'm not sure if this is required but looks like this should be indented?

Reply:
Hi @rtrieu ! Forgot to mention I tested the snippet and it works as expected, since this is a JSON file, and this is just one long string (encapsulated by " " ) , I think what we are seeing here is just the word wrapping, so no it doesn't affect functionality :)

Reply:
Oh silly me, thanks for explaining!